### ISSUE TITLE
Accessibility is 38/100 for our start

### ISSUE THREAD

**Issue by nicknist** [Alkiiro/refactor-tractor#2]
Looks like there's three main parts to this (maybe make sub-issues): 
- Background and Foreground colors don't contrast enough
- html element does not have language tags
- image elements do not have alt tags
